src/Controls/src/Core/Handlers/Shell/ShellHandler.Windows.cs

```diff
@@ -127,6 +129,15 @@ public static void MapFlyoutBackground(ShellHandler handler, Shell view)
 					view.FlyoutBackgroundColor?.AsPaint());
 		}
 
+		internal static void MapFlyoutBackgroundImage(ShellHandler handler, Shell view)
+		{
+			var provider = handler.GetRequiredService<IImageSourceServiceProvider>();
```

Comment:
Not mandatory but could create an extension method like `UpdateFlyoutBackgroundImage` and move this logic there.